pkg/api/v1/conversion.go

```diff
@@ -324,6 +324,11 @@ func Convert_api_PodTemplateSpec_To_v1_PodTemplateSpec(in *api.PodTemplateSpec,
 	} else {
 		delete(out.Annotations, PodInitContainersAnnotationKey)
 	}
+
+	// TODO: when seccomp moves to beta the PSC and SC should get the
+	// SeccompProfile field and this can be removed
+	internalConvertApiSeccompToV1Seccomp(&in.Spec, &out.ObjectMeta)
```

Comment:
Yeah, I left it with this naming convention since it is not a straight type to type conversion